learning_resources_search/serializers.py

```diff
@@ -457,6 +457,25 @@ def get_previous(self, instance) -> str | None:
     def get_count(self, instance) -> int:
         return instance.get("hits", {}).get("total", {}).get("value")
 
+    def get_metadata(self, instance) -> SearchResponseMetadata:
+        return {
+            "aggregations": _transform_aggregations(instance.get("aggregations", {})),
+            "suggest": _transform_search_results_suggest(instance),
+        }
+
+
+class PercolateQuerySerializer(serializers.ModelSerializer):
+    class Meta:
```

Comment:
Maybe add a simple docstring like "Serializer for PercolateQuery objects"

Reply:
I didn't update this code - it's only showing up as new because stuff around it moved. But I'll add a docstring